clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -13399,7 +13399,7 @@ def err_bounds_safety_terminated_by_wrong_initializer_kind : Error<
   "string literal or an initializer list">;
 def err_bounds_safety_terminated_by_terminator_mismatch : Error<
   "array '%0' with '__terminated_by' attribute is initialized with an "
-  "incorrect terminator (expected: %1; got %2)">;
+  "incorrect terminator (expected: %quoted1; got %quoted2)">;
```

Comment:
This seems new. Looking at the implementation it looks like this just adds single quotes around the argument. Presumably this is related to https://github.com/llvm/llvm-project/pull/134769#issuecomment-2822234053 and now we have double single quotes in the outputted diagnostics?